hasher-matcher-actioner/hmalib/lambdas/api/api_root.py

```diff
@@ -161,28 +202,47 @@ def gen_matches() -> t.List[MatchesResult]:
             "signal_id": record.signal_id,
             "signal_source": record.signal_source,
             "updated_at": record.updated_at.isoformat(),
-            "reactions": "TODO",
+            "reactions": "Mocked",
         }
         for record in records
     ]
 
 
+class MatchDetailsMetaData(t.TypedDict):
+    type: str
+    tags: t.List[str]
+    status: str
+    opinions: t.List[str]
+
+
 class MatchDetailsResult(t.TypedDict):
     content_id: str
     content_hash: str
     signal_id: t.Union[str, int]
     signal_hash: str
     signal_source: str
     updated_at: str
+    meta_data: MatchDetailsMetaData
+    actions: t.List[str]
 
 
-def gen_match_details(content_id: str) -> t.List[MatchDetailsResult]:
+def get_match_details(content_id: str) -> t.List[MatchDetailsResult]:
     if not content_id:
         return []
     table = dynamodb.Table(DYNAMODB_TABLE)
     records = PDQMatchRecord.get_from_content_id(
         table, f"{IMAGE_FOLDER_KEY}{content_id}"
     )
+    # TODO these mocked metadata should either be added to
+    # PDQMatchRecord or some other look up in the data model
+    MOCKED_METADATA: MatchDetailsMetaData = {
+        "type": "HASH_PDQ",
```

Comment:
Nit: TypedDicts allow this:

```
  MatchDetailsMetaData(type="HASH_PDQ", tags=[...])
```

Which means you don't have to add a type for them.